<dec f='tensorflow/tensorflow/python/lib/core/py_exception_registry.h' l='68' type='std::map&lt;TF_Code, PyObject *&gt;'/>
<use f='tensorflow/tensorflow/python/lib/core/py_exception_registry.cc' l='34' u='m' c='_ZN10tensorflow19PyExceptionRegistry4InitEP7_object'/>
<use f='tensorflow/tensorflow/python/lib/core/py_exception_registry.cc' l='45' u='m' c='_ZN10tensorflow19PyExceptionRegistry6LookupE7TF_Code'/>
<use f='tensorflow/tensorflow/python/lib/core/py_exception_registry.cc' l='45' u='m' c='_ZN10tensorflow19PyExceptionRegistry6LookupE7TF_Code'/>
<use f='tensorflow/tensorflow/python/lib/core/py_exception_registry.cc' l='47' u='m' c='_ZN10tensorflow19PyExceptionRegistry6LookupE7TF_Code'/>
<offset>0</offset>
<doc f='tensorflow/tensorflow/python/lib/core/py_exception_registry.h' l='67'>// Maps error codes to the corresponding Python exception type.</doc>
